test/stepDefinitions/orNotEqualsSteps.js

```diff
@@ -0,0 +1,47 @@
+let CreateCaseWizardPage = require("../pageObjects/createCaseWizardPage");
```

Comment:
after changes I would move all steps to create-caseSteps.js as that file deals with all the form pages interactions for creating cases - you will also find reusable steps like I have listed in some of the comments there too